packages/client/src/components/interface/Navigation.tsx

```diff
@@ -0,0 +1,648 @@
+  menuGroup: 'menuGroup'
+}
+
+interface IUSER_SCOPE {
+  [key: string]: string[]
+}
+
+const USER_SCOPE: IUSER_SCOPE = {
+  FIELD_AGENT: [
+    TAB_ID.inProgress,
+    TAB_ID.sentForReview,
+    TAB_ID.requireUpdates,
+    GROUP_ID.applicationGroup
+  ],
+  REGISTRATION_AGENT: [
+    TAB_ID.inProgress,
+    TAB_ID.readyForReview,
+    TAB_ID.sentForUpdates,
+    TAB_ID.sentForApproval,
+    TAB_ID.readyToPrint,
+    TAB_ID.performance,
+    TAB_ID.team,
+    GROUP_ID.applicationGroup,
+    GROUP_ID.menuGroup
+  ],
+  LOCAL_REGISTRAR: [
+    TAB_ID.inProgress,
+    TAB_ID.readyForReview,
+    TAB_ID.sentForUpdates,
+    TAB_ID.readyToPrint,
+    TAB_ID.performance,
+    TAB_ID.team,
+    GROUP_ID.applicationGroup,
+    GROUP_ID.menuGroup
+  ],
+  LOCAL_SYSTEM_ADMIN: [TAB_ID.performance, TAB_ID.team, GROUP_ID.menuGroup],
+  NATIONAL_SYSTEM_ADMIN: [
+    TAB_ID.performance,
+    TAB_ID.team,
+    TAB_ID.config,
+    GROUP_ID.menuGroup
+  ]
+}
+
+interface ICount {
+  inProgress?: number
+  readyForReview?: number
+  sentForReview?: number
+  requiresUpdate?: number
+  sentForUpdates?: number
+  sentForApproval?: number
+  externalValidation?: number
+  readyToPrint?: number
+}
+
+interface IUserInfo {
+  name: string
+  role: string
+  avatar: JSX.Element
+}
+
+interface IProps {
+  count?: ICount
+  enableMenuSelection?: boolean
+  navigationWidth?: number
+  menuCollapse?: () => void
+  userInfo?: IUserInfo
+}
+
+interface IDispatchProps {
+  goToFieldAgentHomeTab: typeof goToFieldAgentHomeTabAction
+  goToRegistrarHomeTab: typeof goToRegistrarHomeTab
+  goToConfigAction: typeof goToConfig
+  goToPerformanceHomeAction: typeof goToPerformanceHome
+  goToOperationalReportAction: typeof goToOperationalReport
+  goToTeamSearchAction: typeof goToTeamSearch
+  goToTeamUserListAction: typeof goToTeamUserList
+  redirectToAuthentication: typeof redirectToAuthentication
+  goToSettings: typeof goToSettings
+}
+
+interface IStateProps {
+  draftApplications: IApplication[]
+  applicationsReadyToSend: IApplication[]
+  userDetails: IUserDetails | null
+  activeMenuItem: string
+  workqueue: IWorkqueue
+  storedApplications: IApplication[]
+}
+
+type IFullProps = IProps &
+  IStateProps &
+  IDispatchProps &
+  IntlShapeProps & { theme: ITheme } & RouteComponentProps<{ tabId: string }>
+
+const TAB_LABEL = {
+  inProgress: 'In Progress',
+  readyForReview: 'Ready for review',
+  sentForReview: 'Sent for review',
+  requiresUpdate: 'Requires Updates',
+  sentForUpdates: 'Sent for updates',
+  sentForApproval: 'Sent for approval',
+  externalValidation: 'In external Validation',
+  readyToPrint: 'Ready to print ',
+  application: 'Application',
+  performance: 'Performance',
+  team: 'Team',
+  configuration: 'Configuration',
+  certificatesConfiguration: 'Certificates',
+  applicationSettings: 'Application Settings',
+  settings: 'Settings',
+  logout: 'Logout'
+}
+
+const goToPerformanceView = (props: IFullProps) => {
```

Comment:
What do you think about having these two functions in `packages/client/src/navigation/index.ts` and having the function signatures just as

```
const goToPerformanceView = (userDetails) => {
const goToTeamView = (userDetails) => {
``` 

as there they could directly refer to the action creators you're using internally here?